src/hooks/useRHHCConfig.tsx

```diff
@@ -39,6 +40,8 @@ export default function useRHHCConfig(): Config {
         appLanguageToRHHCLanguageMap[language] ?? LanguageCodeEnum.Fi,
       components: {
         ...rhhcDefaultConfig.components,
+        A: ({ href, ...props }) => <Link to={href || ''} {...props} />,
+        Link: ({ href, ...props }) => <Link to={href || ''} {...props} />,
```

Comment:
```suggestion
        A: ({ href, ...props }) => <Link to={href ?? ''} {...props} />,
        Link: ({ href, ...props }) => <Link to={href ?? ''} {...props} />,
```
To remove the new SonarCloud issues related to https://rules.sonarsource.com/typescript/RSPEC-6606/ i.e. "Nullish coalescing should be preferred"